packages/ember-routing/tests/location/util_test.js

```diff
@@ -84,7 +84,52 @@ QUnit.test("Feature-detecting the history API", function() {
   equal(supportsHistory("", { pushState: true }), true, "returns true if not Android Gingerbread and history.pushState exists");
   equal(supportsHistory("", {}), false, "returns false if history.pushState doesn't exist");
   equal(supportsHistory("", undefined), false, "returns false if history doesn't exist");
-  equal(supportsHistory("Mozilla/5.0 (Linux; U; Android 2.3.5; en-us; HTC Vision Build/GRI40) AppleWebKit/533.1 (KHTML, like Gecko) Version/4.0 Mobile Safari/533.1", { pushState: true }),
-                        false, "returns false if Android Gingerbread stock browser claiming to support pushState");
+
+  equal(
+    supportsHistory(
+      "Mozilla/5.0 (Linux; U; Android 2.3.5; en-us; HTC Vision Build/GRI40) AppleWebKit/533.1 (KHTML, like Gecko) Version/4.0 Mobile Safari/533.1",
+      { pushState: true }
+    ),
+    false,
+    "returns false if Android 2.x stock browser (not Chrome) claiming to support pushState"
+  );
+
+  equal(
+    supportsHistory(
+      "Mozilla/5.0 (Linux; U; Android 4.0.3; nl-nl; GT-N7000 Build/IML74K) AppleWebKit/534.30 (KHTML, like Gecko) Version/4.0 Mobile Safari/534.30",
+      { pushState: true }
+    ),
+    false,
+    "returns false for Android 4.0.x stock browser (not Chrome) claiming to support pushState"
+  );
+
+  equal(
+    supportsHistory(
+      "Mozilla/5.0 (Linux; U; Android 20.3.5; en-us; HTC Vision Build/GRI40) AppleWebKit/533.1 (KHTML, like Gecko) Version/4.0 Mobile Safari/533.1",
+      { pushState: true }
+    ),
+    true,
+    "returns true if Android version begins with 2, but is greater than 2"
+  );
+
+  equal(
+    supportsHistory(
+      "Mozilla/5.0 (Linux; Android 4.0.4; Galaxy Nexus Build/IMM76B) AppleWebKit/535.19 (KHTML, like Gecko) Chrome/18.0.1025.133 Mobile Safari/535.19",
+      { pushState: true }
+    ),
+    true,
+    "returns true for Chrome (not stock browser) on Android 4.0.x"
+  );
+  
```

Comment:
Weird.  JSCS fails here because it thinks you have two newlines (which is disallowed).  I think that is because the comments are stripped (leaving two empty lines) when checking for style, then re-integrated for the error message.

Can you add `// jscs:disable` and `// jscs:enable` comments surrounding this whole test?